api/app/route.ts

```diff
@@ -3,17 +3,42 @@ import { NextRequest, NextResponse } from "next/server";
 
 export const runtime = "edge";
 
+type exampleResponseObject = {
```

Comment:
```suggestion
type ExampleResponseObject = {
```

Super minor thing, but types are usually capitalized